src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final String name;
```

Comment:
Do you actually need it? See next comments and remove it in all other cases.

Reply:
done